it/src/test/java/apoc/it/core/TriggerEnterpriseFeaturesTest.java

```diff
@@ -435,32 +433,20 @@ public void stop() {
     }
 }
 
-record CreateTriggers(Driver driver, String db, int iterations) implements Runnable {
+record CreateAndDropTriggers(Driver driver, String db, int iterations) implements Runnable {
     public static final String CreateTrigger = "call apoc.trigger.install($db, $name, $trigger,{})";
-
-    @Override
-    public void run() {
-        try (final var session = driver.session(forDatabase(SYSTEM_DATABASE_NAME))) {
-            for (int i = 0; i < iterations; ++i) {
-                final var name = "temp-trigger-" + i;
-                session.run(CreateTrigger, Map.of("db", db, "name", name, "trigger", "RETURN 1"))
-                        .consume();
-            }
-        }
-    }
-}
-
-record DropRandomTriggers(Driver driver, String db, int iterations) implements Runnable {
```

Comment:
Note, before we dropped and triggers at random concurrently to creating them. I see no reason why that should not be possible. But since install trigger failed sometimes (EntityNotFound) I change it here to not drop and install concurrently.

Reply:
It's not the clearest test, but it was really good at reproducing a previous bug. Your statements are correct. The randomness does not serve that much purpose now. I guess I kept it to have some triggers live on for different amounts of time in case that makes a difference.